src/components/formCovid/index.tsx

```diff
@@ -10,11 +10,13 @@ const FormCovid = (): FC => {
   const [date, setDate] = useState();
   const [state, setState] = useState();
   const [user, setUser] = useState();
+  const [file, setFile] = useState();
   const { solidStore } = useStores();
 
   const handleSubmit = async (e): Promise<any> => {
     e.preventDefault();
     await solidStore.createCovidFile(date, certificaat, session);
+    await solidStore.handleFiles(file, session);
     await solidStore.grantAccesToCovidFile(session, user);
     setState(solidStore.status);
```

Comment:
I think this state is unnecessary, because the `solidStore.status` is the same value? So I don't think you need a separate state for this. 

When setting the status in the solidStore, the useStores() should trigger a rerender i guess? Correct me if i'm wrong :)

Reply:
The solidstore status just gives information that the files have been processed. Its just a basic string :)

Reply:
Yes, but you just show the status in de frontend. Why do you need a separate state for that?

Reply:
Yes indeed, i didn't realise your where talking about the setStatus 😄.  Should be fixed now